Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -1004,7 +1004,8 @@ describe(
     });
 
     it("verify memory usage statistics", function () {
-      // Calculations in Batched3DModel3DTileContentSpec, minus uvs
+      // 10 buildings, 36 ushort indices and 24 vertices per building, 6 float components (position, normal) and 1 uint component (batchId) per vertex.
+      // 10 * ((24 * (6 * 4 + 1 * 4)) + (36 * 2)) = 7440
```

Comment:
can we wrap this comment + change the outer brackets for readability?
```suggestion
      // 10 buildings, 36 ushort indices and 24 vertices per building, 6 float components (position, normal)
      // and 1 uint component (batchId) per vertex.
      // 10 * [(24 * (6 * 4 + 1 * 4) + (36 * 2)] = 7440
```